src/test/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/service/JiraDeploymentInfoSenderImplTest.java

```diff
@@ -352,6 +355,7 @@ private JiraDeploymentInfoRequest createRequest(
                 null,
                 Collections.emptySet(),
                 Boolean.FALSE,
+                Collections.emptySet(),
```

Comment:
Can you add a test that covers the non-empty case?

Reply:
> @PatrikSchalin Changes look good. I have left a few minor comments.

Great! I'll get on the minor changes.